diff --git a/kubernetes/kserve/build_image.sh b/kubernetes/kserve/build_image.sh
index b644dc37..200a431d 100755
--- a/kubernetes/kserve/build_image.sh
+++ b/kubernetes/kserve/build_image.sh
@@ -1,8 +1,8 @@
 #!/bin/bash

 MACHINE=cpu
-DOCKER_TAG="pytorch/torchserve-kfs:latest"
-BASE_IMAGE="pytorch/torchserve:latest-cpu"
+DOCKER_TAG="intel/torchserve:latest-kfs"
+BASE_IMAGE="intel/torchserve:latest"
 DOCKER_FILE="Dockerfile"
 BUILD_NIGHTLY=false
 USE_CUSTOM_TAG=false
@@ -57,4 +57,4 @@ fi
 cp ../../frontend/server/src/main/resources/proto/*.proto .
 cp -r ../../third_party .

-DOCKER_BUILDKIT=1 docker build --file "$DOCKER_FILE" --build-arg BASE_IMAGE=$BASE_IMAGE -t "$DOCKER_TAG" .
+DOCKER_BUILDKIT=1 docker build --file "$DOCKER_FILE" --build-arg http_proxy=${http_proxy} --build-arg https_proxy=${https_proxy} --build-arg BASE_IMAGE=$BASE_IMAGE -t "$DOCKER_TAG" .
